lib/common_test/include/ct.hrl

```diff
@@ -39,5 +39,4 @@
 %% Backward compatibility for test_server test suites.
 %% DO NOT USE IN NEW TEST SUITES.
 -define(line,).
--define(t,test_server).
```

Comment:
such definition is copy/pasted(?) in several other files (e.g. disk_log_SUITE.erl) and seems unused. can you remove them as well in this PR?